sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2265,25 +2266,24 @@ test_suite(
     # does not matter but opentitan_functest() is unhappy if we don't provide one.
     # Since execution in the ROM is disabled, bootstrap is not possible so we need
     # to make sure that the test does not try to bootstrap
-    opentitan_functest(
+    opentitan_test(
```

Comment:
nit: reference to `opentitan_functest` in above comment

Reply:
Fixed.